COPYRIGHT.txt

```diff
@@ -131,6 +131,11 @@ Comment: Intel ASSAO and related files
 Copyright: 2016, Intel Corporation
 License: Expat
 
+Files: ./servers/rendering/renderer_rd/shaders/taa_resolve.glsl
+Comment: Temporal Anti-Aliasing reslove implementation
```

Comment:
There's a typo here: "reslove"

Reply:
Fixed in https://github.com/godotengine/godot/pull/61815.